workflow/scripts/goatools-go-enrichment-analysis.py

```diff
@@ -52,6 +51,26 @@
 
 goea_results_all = goeaobj.run_study(sig_genes["ens_gene"].tolist())
 
+#run one time to initialize
+GO_items = []
+
+temp = goeaobj.ns2objgoea['BP'].assoc
+for item in temp:
+    GO_items += temp[item]
+    
+
+temp = goeaobj.ns2objgoea['CC'].assoc
+for item in temp:
+    GO_items += temp[item]
+    
+
+temp = goeaobj.ns2objgoea['MF'].assoc
+for item in temp:
+    GO_items += temp[item]
```

Comment:
Simplification, please check:
```suggestion
go_items = [val for cat in ["BP", "CC", "MF"] for item, val in goeaobj.nsobjgoea[cat].assoc.items()]
```
Also, note that variable names in Python have to be in snake_case (all lower).

Reply:
Noted. Changed